.all-contributorsrc

```diff
@@ -1,414 +1,346 @@
-        "fr_FR"
-      ],
-      "login": "mattnounet",
-      "name": "mattnounet",
-      "profile": "https://github.com/mattnounet"
-    },
-    {
-      "login": "scottyboii",
-      "name": "scottybbooyy",
-      "avatar_url": "https://avatars.githubusercontent.com/u/32883419?v=4",
-      "profile": "https://github.com/scottyboii",
-      "contributions": [
-        "doc",
-        "en_GB"
-      ]
-    },
-    {
-      "login": "rikivideo9",
-      "name": "Riccardo Tocchetto",
-      "avatar_url": "https://avatars.githubusercontent.com/u/21045919?v=4",
-      "profile": "https://github.com/rikivideo9",
-      "contributions": [
-        "doc",
-        "translation",
-        "it_IT"
-      ]
-    },
-    {
-      "login": "Gooochy",
-      "name": "Gooochy",
-      "avatar_url": "https://avatars.githubusercontent.com/u/85019809?v=4",
-      "profile": "https://github.com/Gooochy",
-      "contributions": [
-        "translation",
-        "en_GB"
-      ]
-    },
-    {
-      "login": "02Miki",
-      "name": "02Miki",
-      "avatar_url": "https://avatars.githubusercontent.com/u/35926600?v=4",
-      "profile": "https://github.com/02Miki",
-      "contributions": [
-        "translation",
-        "it_IT"
-      ]
-    },
-    {
-      "login": "deoxis9001",
-      "name": "deoxis9001",
-      "avatar_url": "https://avatars.githubusercontent.com/u/638243?v=4",
-      "profile": "https://github.com/deoxis9001",
-      "contributions": [
-        "doc",
-        "translation",
-        "fr_FR"
-      ]
-    },
-    {
-      "login": "Kemon22",
-      "name": "Kemon58",
-      "avatar_url": "https://avatars.githubusercontent.com/u/31404125?v=4",
-      "profile": "https://www.operatorratunkowy.pl/profile/304057",
-      "contributions": [
-        "doc",
-        "translation",
-        "pl_PL"
-      ]
-    },
-    {
-      "login": "Aapeli1304",
-      "name": "Aapeli1304",
-      "avatar_url": "https://avatars.githubusercontent.com/u/102477892?v=4",
-      "profile": "https://github.com/Aapeli1304",
-      "contributions": [
-        "translation",
-        "fi_FI"
-      ]
+    "badgeTemplate": "[![All Contributors](https://img.shields.io/badge/all_contributors-<%= contributors.length %>-orange.svg?style=flat-square)](#contributors)",
+    "commitConvention": "gitmoji",
+    "contributors": [
+        {
+            "avatar_url": "https://avatars1.githubusercontent.com/u/27238105?v=4",
+            "contributions": [
+                "code",
+                "doc",
+                "financial",
+                "infra",
+                "maintenance",
+                "mentoring",
+                "projectManagement",
+                "question",
+                "review",
+                "de_DE"
+            ],
+            "login": "jxn-30",
+            "name": "Jan",
+            "profile": "https://github.com/jxn-30"
+        },
+        {
+            "avatar_url": "https://avatars0.githubusercontent.com/u/30264441?v=4",
```

Comment:
New linter rule?

Reply:
nopes, that file is not linted at all. allcontributors seem to have changed their output format.